requirements.txt

```
@@ -1,5 +1,6 @@
 render-python>=2.0.1
-argschema>=1.15.17
+marshmallow==2.18.1
```

```suggestion
marshmallow<3.0
```

I would rather do marshmallow <3 since there are many versions that seem to work with this (I even think that 3+ is compatible with the current argschema release, but I guess that's nor guaranteed.)

---

confirmed, that works